scipy/spatial/distance.py

```diff
@@ -888,7 +886,15 @@ def seuclidean(u, v, V):
     """
     Return the standardized Euclidean distance between two 1-D arrays.
 
-    The standardized Euclidean distance between `u` and `v`.
+    Computes the standardized Euclidean distance. The standardized
```

Comment:
This sentence is redundant. "Returns x. Computes x .

Reply:
I have removed the redundant sentence. Thank you for the feedback.